pandas/io/gbq.py

```diff
@@ -682,6 +686,13 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs: Arbitrary keyword arguments
```

Comment:
@jreback Hmm, in this case I can use ´´´read_gbq´´´  for loading files, isn't it? I mean:
´´´
config = {'load' : ....}
´´´
and function ´´´read_gbq´´´ could use not only for reading queries and parameter ´´´query´´´ seems unnecessary. Could you give me some more motivation about configuration parameter? 
I think bigquery jobs like ´´´load´´´, ´´´copy´´´ are outside of pandas philosophy?